pkg/config/assets/service_config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
i think this is leftover from a bad merge. we don't use this file anymore. the one in the `assets/services` dir is the new style